README.asciidoc

```diff
@@ -29,24 +29,23 @@ See https://github.com/mawww/golf for Kakoune solutions to vimgolf challenges,
 regularly beating the best Vim solution.
 
 See the link:doc/design.asciidoc[design document] for more information on
-Kakoune philosophy and design.
+Kakoune's philosophy and design.
 
 :numbered:
 
 Introduction
 ------------
 
 Kakoune is a code editor that implements Vi's "keystrokes as a text editing
-language" model. As it's also a modal editor, it is somewhat similar to the
+language" model. As it is also a modal editor, it is somewhat similar to the
 Vim editor (after which Kakoune was originally inspired).
 
-Kakoune can operate in two modes, normal and insertion. In insertion mode,
+Kakoune can operate in two modes: *normal* and *insertion*. In insertion mode,
 keys are directly inserted into the current buffer. In normal mode, keys
 are used to manipulate the current selection and to enter insertion mode.
 
-Kakoune has a strong focus on interactivity, most commands provide immediate
-and incremental results, while still being competitive (as in keystroke count)
-with Vim.
+Kakoune has a strong focus on interactivity. Most commands provide immediate
+and incremental results, while having nearly the same keystroke count as vim.
```

Comment:
I think the "competitive" was pretty strong, we should probably keep it.
(also Vim over vim is consistent with how we spell Kakoune).

Reply:
Okay, changes made in new commit.

Reply:
`git commit --amend` is fine but it looks like you didn't push an actual content change but only a redundant merge commit.
When you push a rewritten history you need `git push --force`.
If you don't use `--force`, git will tell you to pull (the old version) first.
That's actually misleading advice here, if you follow it, git will create a merge commit that combines the local and the remote branches.
For this simple change we don't need merge commits, a single commit with all changes is enough. A quick `git rebase -i` drops merge commits by default.

Reply:
Okay! Done. How's it look?

I still want to amend the rest of the README. I sent this PR early because I wasn't sure how the response would be. I could amend this PR or I can make a new one for the second half of the README.

Reply:
Thanks, the changes look good to me (except for one thing I'll comment on).

> I could amend this PR or I can make a new one for the second half of the README.

If you'll add similar changes, then amending is a good idea because it makes the next reviewer more efficient.